Makefile

```diff
@@ -449,7 +476,8 @@ init-swarm: swarm-init swarm-networks swarm-configs swarm-secrets
 kubernetes:
 	$(DIR)/bin/kompose --file $(DIR)/lib/kubernetes/docker-compose.yml \
 		$(foreach MODULE, $(CANDIG_MODULES), --file $(DIR)/lib/$(MODULE)/docker-compose.yml) \
-		up
+		convert
```

Comment:
please evaluate whether or not convert is required, we shouldn't need to generate kubernetes manifests

Reply:
this is a mistake - should not be part of this PR